### PR TITLE
Instrumentation.AWSLambda 1.2.0-beta.1 release

### PR DESCRIPTION
**Pull request by rypdal**
Instrumentation.AWSLambda 1.2.0-beta.1 release

### CODE REVIEW

File: src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.2.0-beta.1
+
+Released 2023-Aug-03
+
```
Comment (Oberon00) [inline src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md:8]:
```suggestion

* BREAKING: `AddAWSLambdaConfigurations` no longer removes all existing resource attributes
```

**Review by Oberon00 — APPROVED** [review]

File: src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,13 @@
 
 ## Unreleased
 
-* Change dependency from `OpenTelemetry.Contrib.Extensions.AWSXRay` to `OpenTelemetry.Extensions.AWS`
-  ([#1289](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1289))
+## 1.2.0-beta.1
+
+Released 2023-Aug-03
+
+* BREAKING: `AddAWSLambdaConfigurations` no longer removes all existing resource attributes
```
Comment (Kielek) [inline src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md:9]: To long lines - please markdown lint check.

File: src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md [hunk 3]
```diff
@@ -2,8 +2,13 @@
...
+* BREAKING: Change dependency from `OpenTelemetry.Contrib.Extensions.AWSXRay` to `OpenTelemetry.Extensions.AWS`
+  ([#1289](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1289)). This now requires at least OpenTelemetry 1.5.1.
```
Comment (Kielek) [inline src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md:11]: See @utpilla comment https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1289#discussion_r1282288389

File: src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md (hunk 2 above)
Comment (rypdal) [reply src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md]: I know. Fixed already.

File: src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md (hunk 3 above)
Comment (rypdal) [reply src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md]: Fixed.

Comment (Oberon00) [reply src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md]: I think this is relevant enough for users of this library to keep the repetition it here (and the reason for adding BREAKING)

**Review by Kielek — APPROVED** [review]

File: src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md [hunk 4]
```diff
@@ -2,8 +2,16 @@
 
 ## Unreleased
 
-* Change dependency from `OpenTelemetry.Contrib.Extensions.AWSXRay` to `OpenTelemetry.Extensions.AWS`
-  ([#1289](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1289))
+## 1.2.0-beta.1
+
+Released 2023-Aug-03
```
Comment (Kielek) [inline src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md:7]:
```suggestion
Released 2023-Aug-07
```

**Comment by Kielek** [comment]
https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AWSLambda/1.2.0-beta.1

**Pull request merged**